content/en/serverless/aws_lambda/instrumentation/ruby.md

```diff
@@ -16,19 +16,16 @@ further_reading:
 aliases:
     - /serverless/datadog_lambda_library/ruby/
     - /serverless/installation/ruby
+    - /serverless/aws_lambda/instrumentation/ruby
```

Comment:
```suggestion
    - /serverless/aws_lambda/installation/ruby
```

Reply:
I _think_ this is what's intended, but let me know if I'm missing something here